src/concurrent.cr

```diff
@@ -128,21 +128,66 @@ macro spawn(call, *, name = nil)
   {% end %}
 end
 
+# Runs the commands passed as arguments concurrently (in Fibers) and waits
+# for them to finish.
+#
+# ```
+# def say(word)
+#   puts word
+# end
+#
+# # Will print out the three words concurrently
+# parallel(
+#   say("concurrency"),
+#   say("is"),
+#   say("easy")
+# )
+# ```
+#
+# Can also be used to conveniently collect the return values of the
+# concurrent operations.
+#
+# ```
+# def concurrent_job(word)
+#   word
+# end
+#
+# a, b, c =
+#   parallel(
+#     concurrent_job("concurrency"),
+#     concurrent_job("is"),
+#     concurrent_job("easy")
+#   )
+#
+# puts a # => "concurrency"
+# puts b # => "is"
+# puts c # => "easy"
+# ```
+#
+# Exceptions raised within the concurrent operations will be re-raised
+# in the parent thread.
```

Comment:
`parent fiber`

Reply:
This should also explain what happens with the other tasks if one raises an exception or what happens if more than one tasks raises.